packages/react-app/src/pages/manage.tsx

```diff
@@ -33,8 +28,6 @@ const Manage = () => {
   const { isLoading, data } = GetGiveaways();
   const [activeTab, setActiveTab] = useState(Tabs.Active);
   const [countdownGiveaway, setCountdownGiveaway] = useState<Giveaway | null>();
-  const [winnerGiveaways, setWinnerGiveaways] = useState<Giveaway[]>([]);
-  const [rejectedGiveaways, setRejectedGiveaways] = useState<Giveaway[]>([]);
   const [error, setError] = useState(false);
 
   const giveaways = useMemo(() => {
```

Comment:
In the future this filtering should be done in the API, using query parameters to filter the results